src/components/StokesAnalysis/StokesAnalysisComponent.tsx

```diff
@@ -114,10 +117,17 @@ export class StokesAnalysisComponent extends React.Component<WidgetProps> {
                 let progressString = "";
                 const currentData = this.plotData;
                 if (currentData && isFinite(currentData.qProgress) && isFinite(currentData.uProgress)) {
-                    let minProgress = Math.min(currentData.qProgress, currentData.uProgress);
+                    const minProgress = Math.min(currentData.qProgress, currentData.uProgress);
+                    let currentProgress = "";
+                    if (minProgress === currentData.qProgress) {
+                        currentProgress = "Qz";
+                    } else {
+                        currentProgress = "Uz";
+                    }
                     if (minProgress < 1) {
-                        progressString = `[${(minProgress * 100).toFixed(0)}% complete]`;
+                        progressString = `[${(minProgress * 100).toFixed(0)}% complete ${currentProgress}]`;
```

Comment:
not sure if the user needs to know this. `Qz` an `Uz` are not standard terms.

Reply:
I agree.  The labels for the Stokes in the Stokes widget should just be Q, U rather than Qz, Uz.

Reply:
on that note, @panchyo0, we should make sure that the legend displays the colors nicely. In order to do this, we need to set the `backgroundColor` of the plots as well as the border color. I've just added a commit to this branch that does this.